### ISSUE TITLE
Flow-typed install fails since today if ./flow-typed/npm is not deleted

### ISSUE THREAD

**Issue by HyperBrain** [flowtype/flow-typed#1310]
Since today, a install of the type libraries fails if you do it in an already existing project.

```
 node ./node_modules/flow-typed/dist/cli.js install

• Found 86 dependencies in package.json to install libdefs for. Searching...
• rebasing flow-typed cache...
UNCAUGHT ERROR: Error: flow_v0.47.x <=v0.54.x: Unexpected trailing characters:  <=v0.54.x
    at validationError (xxxxxx\node_modules\flow-typed\dist\lib\validationErrors.js:14:11)
    at parseDirString (xxxxxx\node_modules\flow-typed\dist\lib\flowVersion.js:155:45)
    at _callee2$ (xxxxxx\node_modules\flow-typed\dist\lib\npm\npmLibDefs.js:543:146)
    at tryCatch (xxxxxx\node_modules\regenerator-runtime\runtime.js:65:40)
```

This happens on Windows as well as on our Linux build server.

It seems that one of the recent commits triggers the behavior.
However, if I delete the `./flow-typed/npm` directory it succeeds.

**Comment by dreid** [comment]
This looks like a dup of #1151 which should be fixed by #1311.

**Comment by marudor** [comment]
I've released v2.2 with the fix by #1311
